app/workers/delivery_request_worker.rb

```diff
@@ -49,6 +49,7 @@ def increment_rate_limiter
     rate_limiter.add("delivery_request")
   end
 
+  # Notify shares the same rate limit of 350r/s
```

Comment:
Instead of repeating the number, can we point to [the dev doc](https://github.com/alphagov/govuk-developer-docs/pull/2639) for more info? We should do that so any changes here are kept in sync.

Reply:
Good point!